<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GinyUdpTramet</class>
 <widget class="QWidget" name="GinyUdpTramet">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>591</width>
    <height>193</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GinyUdpTramet</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_4">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QRadioButton" name="radioButtonSingle">
           <property name="text">
            <string>Sin&amp;gle IP</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonSingleDN">
           <property name="text">
            <string>Single D&amp;N</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QRadioButton" name="radioButtonBroadcast">
           <property name="text">
            <string>&amp;Broadcast</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="10,0">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>&amp;IP: </string>
               </property>
               <property name="buddy">
                <cstring>leIP</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leIP">
               <property name="font">
                <font>
                 <family>Monospace</family>
                </font>
               </property>
               <property name="inputMask">
                <string>000.000.000.000</string>
               </property>
               <property name="text">
                <string>127.0.0.1</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>&amp;DN: </string>
               </property>
               <property name="buddy">
                <cstring>leIP</cstring>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLineEdit" name="leDN">
               <property name="font">
                <font>
                 <family>Monospace</family>
                </font>
               </property>
               <property name="inputMask">
                <string/>
               </property>
               <property name="text">
                <string>esp8266-dam-xx.local</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Tx &amp;Port: </string>
             </property>
             <property name="buddy">
              <cstring>lePort</cstring>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="lePort">
             <property name="font">
              <font>
               <family>Monospace</family>
              </font>
             </property>
             <property name="text">
              <string>3334</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Text: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="leText">
         <property name="text">
          <string>i</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btSend">
         <property name="text">
          <string>&amp;Send</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="0,10">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <widget class="QCheckBox" name="cbR">
         <property name="text">
          <string>R</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbG">
         <property name="text">
          <string>G</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbB">
         <property name="text">
          <string>B</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="cbLittleB">
         <property name="text">
          <string>b</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lbInfo">
       <property name="font">
        <font>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Info</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>leText</tabstop>
  <tabstop>btSend</tabstop>
  <tabstop>radioButtonSingle</tabstop>
  <tabstop>radioButtonBroadcast</tabstop>
  <tabstop>leIP</tabstop>
  <tabstop>lePort</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>radioButtonBroadcast</sender>
   <signal>clicked(bool)</signal>
   <receiver>leIP</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>70</x>
     <y>101</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonSingle</sender>
   <signal>clicked(bool)</signal>
   <receiver>leIP</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>52</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>295</x>
     <y>45</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>leText</sender>
   <signal>returnPressed()</signal>
   <receiver>btSend</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>271</x>
     <y>138</y>
    </hint>
    <hint type="destinationlabel">
     <x>506</x>
     <y>138</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonBroadcast</sender>
   <signal>clicked(bool)</signal>
   <receiver>leDN</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>84</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>173</x>
     <y>82</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonSingle</sender>
   <signal>clicked(bool)</signal>
   <receiver>leDN</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>32</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>74</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonSingleDN</sender>
   <signal>clicked(bool)</signal>
   <receiver>leIP</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>74</x>
     <y>54</y>
    </hint>
    <hint type="destinationlabel">
     <x>158</x>
     <y>39</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButtonSingleDN</sender>
   <signal>clicked(bool)</signal>
   <receiver>leDN</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>97</x>
     <y>56</y>
    </hint>
    <hint type="destinationlabel">
     <x>180</x>
     <y>80</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
